lib/lyber_core/base.rb

```diff
@@ -0,0 +1,76 @@
+require 'benchmark'
+require 'active_support'
+
+module LyberCore
+  # The skeleton of robots, a replacement for LyberCore::Robot
+  class Base
```

Comment:
what do you think about BaseRobot as the class name?

Reply:
Seems unnecessary.  If you have a single example of existing code that occupies that namespace, then I would be compelled to rename.  I would opt for something like `LyberCore::Robot::Base`.